src/python/pants/backend/python/util_rules/pex_from_targets_test.py

```diff
@@ -807,13 +807,10 @@ def test_lockfile_requirements_selection(
         mode_files.update({"3rdparty/python/default.lock": setuptools_poetry_lockfile})
     else:
         assert mode == ResolveMode.pex
-
-        requirements = rule_runner.request(Setuptools, []).pex_requirements()
-        assert isinstance(requirements, EntireLockfile)
-        assert isinstance(requirements.lockfile, LockfileContent)
-        mode_files.update(
-            {"3rdparty/python/default.lock": requirements.lockfile.file_content.content}
+        lock_content = importlib.resources.read_binary(
```

Comment:
This was always available to us as a more streamlined way of getting this content for this test. Now we have to use it...